### PR TITLE
Pass the portable pipeline repository URL to the autoqc generic job.

### PR DESCRIPTION
**Pull request by mgcam**
pp_repo_url parameter is read from the study configuration for a
portable pipeline and, if defined, is passed to the autoqc generic job
so that the URL can be captured in the information about the autoqc
generic check, which is saved by the generic result object.

Depends on https://github.com/wtsi-npg/npg_qc/pull/762.

### CODE REVIEW

File: lib/npg_pipeline/product/release/portable_pipeline.pm [hunk 1]
```diff
@@ -92,6 +93,19 @@ sub pp_version {
   return $pp_conf->{$PP_VERSION_KEY};
 }
 
+=head2 pp_repo_url
+
+Given a configurationn hash for the portable pipeline,
```
Comment (nerdstrike) [inline lib/npg_pipeline/product/release/portable_pipeline.pm:98]: s/nn/n/

File: t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ study:
     portable_pipelines:
       - pp_name: "ncov2019-artic-nf"
         pp_version: "cf01166c42a"
+        pp_repo_url: "https://github.com/connor-lab/ncov2019-artic-nf"
```
Comment (nerdstrike) [inline t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml:16]: Is it ok that this URL doesn't reference a particular release? Does pp_version somehow reference a commit?

**Review by nerdstrike — CHANGES_REQUESTED** [review]
Just one typo and a general question

File: t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml (hunk 2 above)
Comment (mgcam) [reply t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml:16]: pp_version is either a commit or a tag. For us in production it was always a tag, while in development/testing it's often a commit.

Comment (nerdstrike) [reply t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml:16]: Very good

Comment (mgcam) [reply t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml:16]: POD extended.

**Review by nerdstrike — APPROVED** [review]

**Pull request merged**